src/Microsoft.Dnx.Tooling/Restore/EmbeddedPackagesWalkProvider.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.Versioning;
+using System.Threading.Tasks;
+using Microsoft.Dnx.Runtime;
+using Microsoft.Dnx.Tooling.Restore.RuntimeModel;
+using NuGet;
+
+namespace Microsoft.Dnx.Tooling
+{
+    internal class EmbeddedPackagesWalkProvider : IWalkProvider
+    {
+        // Based on values from the real Microsoft.NETCore.Platforms, but without 'aot' since DNX doesn't support .NET Native
```

Comment:
"Ahead of Time" as in .NET Native. There are special runtime libraries to be used when building a .NET Native library, but we don't support those in DNX.